extensions/amp-analytics/0.1/variables.js

```diff
@@ -335,6 +335,11 @@ export class VariableService {
           );
         });
       },
+      'SESSION_TIMESTAMP': () => {
+        return this.sessionManagerPromise_.then((sessionManager) => {
+          return sessionManager.getSessionValue(type, SESSION_VALUES.TIMESTAMP);
+        });
+      },
```

Comment:
Please extract a method since callback logic is now duplicated with `SESSION_ID`.

```js
getSessionValue_(analyticsType, key) {
  return this.sessionManagerPromise_.then((sessionManager) => {
    return sessionManager.getSessionValue(analyticsType, key);
  });
}
```

```suggestion
      'SESSION_ID': () =>
        this.getSessionValue_(type, SESSION_VALUES.SESSION_ID),
      'SESSION_TIMESTAMP': () =>
        this.getSessionValue_(type, SESSION_VALUES.TIMESTAMP),
```